bindings/c/library.cpp

```diff
@@ -60,7 +60,7 @@ namespace
 	int add_handle(libtorrent::torrent_handle const& h)
 	{
 		std::vector<libtorrent::torrent_handle>::iterator i = std::find_if(handles.begin()
-			, handles.end(), !boost::bind(&libtorrent::torrent_handle::is_valid, _1));
+			, handles.end(), !std::bind(&libtorrent::torrent_handle::is_valid, _1));
```

Comment:
I'm actually a bit surprised this works. are you sure it means the same thing as with boost::bind? boost has some lambda capabilities which did not make it into the standard, and I would expect this to be one of them.